configuration/configstorage.go

```diff
@@ -0,0 +1,7 @@
+package configuration
+
+// ConfigStorage ...
+type ConfigStorage interface {
+	Get(key Key) (string, error)
```

Comment:
@khalifapro for simplicity can we keep only 1 return value of string and ignore returning the error. If any error we should log it and return empty string value.
This makes our job easier at the consumer side considering we will have to fetch all params for SSM.
For e.g. auth0Clientprovider has 90% of config variables to be retrieved from ssm.